src/stm32/stm32f0_timer.c

```diff
@@ -24,10 +24,16 @@
 #define TIMx TIM2
 #define TIMx_IRQn TIM2_IRQn
 #define HAVE_TIMER_32BIT 1
-#else
+#elif defined(TIM3)
 #define TIMx TIM3
+#ifdef TIM4
+#define TIMx_IRQn TIM3_TIM4_IRQn
+#else
 #define TIMx_IRQn TIM3_IRQn
+#endif
 #define HAVE_TIMER_32BIT 0
+#else
+#error No system timer defined
```

Comment:
I'm lost in `ifdefs`.  What does this do?  Can this block of ifdefs be indented to make it more clear?

This looks like it adds an extra "no system timer defined" path - why?

Reply:
The error was added as a prevention in case I messed something up with the timers definition. I'll change the ifdefs for more clarity.

Reply:
I removed the error case and added indentation. The same issue as with the LPUART2 also applies to the IRQn here.